clients/src/main/java/org/apache/kafka/common/errors/CoordinatorNotAvailableException.java

```diff
@@ -20,24 +20,24 @@
  * The broker returns this error code for consumer metadata requests or offset commit requests if the offsets topic has
  * not yet been created.
```

Comment:
We probably need to update the documentation for the CoordinatorNotAvailable request.